build/release/tmpl/announce.tmpl

```diff
@@ -20,11 +20,30 @@ Title: [ANNOUNCE] Release Apache Kyuubi(Incubating) {release_version}
 Content:
 Hi all,
 
+# (To user@spark.apache.org)
+# Hello Spark Community,
```

Comment:
`Spark Community` ?

Reply:
Yea, we are planning to send this mail to `dev@kyuubi`, `general@incubator`, and `user@spark`.

Reply:
Got